### PR TITLE
dleyna: purge recipes that can be found in meta-oe

### PR DESCRIPTION
**Pull request by GordanM**
As of krogoth branch of meta-openembedded dleyna recipes can be found in
the upstream repository. Meta-bistro dleyna recipes are not being actively
used and as such can be removed.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

### CODE REVIEW

File: layers/multimedia-layer/recipes-multimedia/dleyna/dleyna-git.inc [hunk 1]
```diff
@@ -1,25 +0,0 @@
-SECTION = "multimedia"
-LICENSE = "LGPLv2.1"
-LIC_FILES_CHKSUM = "file://${WORKDIR}/git/COPYING;md5=4fbd65380cdd255951079008b364516c"
-
-DEPENDS += "gupnp gupnp-dlna gupnp-av"
-
-SRC_URI = "git://github.com/01org/${BPN}.git;protocol=https;branch=master"
```
Comment (DunderRoffe) [inline layers/multimedia-layer/recipes-multimedia/dleyna/dleyna-git.inc]: Can not find this recipe on oe:
https://layers.openembedded.org/layerindex/branch/master/recipes/?q=dleyna

File: layers/multimedia-layer/recipes-multimedia/dleyna/dleyna-server/dleyna.service [hunk 2]
```diff
@@ -1,11 +0,0 @@
-[Unit]
-Description=Dleyna server
-After=syslog.target
-After=lightmediascanner.service
-After=rygel.service
-
-[Service]
-ExecStart=/usr/bin/dbus-send --session --print-reply --dest=com.intel.dleyna-server /com/intel/dLeynaServer com.intel.dLeynaServer.Manager.GetServers
```
Comment (DunderRoffe) [inline layers/multimedia-layer/recipes-multimedia/dleyna/dleyna-server/dleyna.service]: This service file does not seem to exist upstream.
http://cgit.openembedded.org/meta-openembedded/tree/meta-multimedia/recipes-multimedia/dleyna/

**Comment by zeenix** [comment]
It's a no-brainer I guess after @DunderRoffe 's comments are addressed.

**Comment by DunderRoffe** [comment]
Ok as long as it is intentional. I misinterpreted the commit message thought dlenya recipes ment all of them not the specifics that do not exist upstream. Maybe the short log should say "remove dlenya recipes / services" instead as we do not really care even if they are upstream?

**Comment by GordanM** [comment]
@DunderRoffe thanks for the explanation, I'll rewrite the commit message to make it more clear.

**Comment by zeenix** [comment]
> As of krogoth branch of meta-openembedded dleyna recipes can be found in
> the upstream repository. 

I had to read it many times to be sure what you mean. An ',' after meta-openembedded, would make it much more readable. :)

> Meta-bistro dleyna recipes are not being actively
> used and as such are not being maintained. 

Same here: after "such". 

>Once the PELUX platform decides
> to use dleyna once again the appendments will be found in the same place.

* Same here: after "again".
* appendments? That's not a word. :)
* Also if you could specify the "same place", that would be even better.
* Yes, I'm a nitpicker. :D

**Comment by GordanM** [comment]
@zeenix fixed. An appendment is indeed a word.

**Comment by zeenix** [comment]
> @zeenix fixed. An appendment is indeed a word.

Thanks. Interesting. For some reason it's marked red here.

**Comment by zeenix** [comment]
Sorry, I'm a bit confused on this part:

> the appendments will be found in the same place:
> 
> layers/multimedia-layer/recipes-multimedia/dleyna/

You mean, they'll still be *added* to the same place? You're removing them so they won't be anywhere to be found? If you mean in the upstream, I don't think the location in upstream is important enough to be mentioned here and will only confuse idiots like me. :)

**Comment by GordanM** [comment]
Quote: "Once the PELUX platform decides
to use dleyna once again, the appendments will be found in the same place:

layers/multimedia-layer/recipes-multimedia/dleyna/"

I am not sure how to make it more clearer :).
Once we start using dleyna as we probably will, we might want to append the recipes, which we will, the appendments will be found in that path. So from that commit message you could historically see why the content changed so drastically.

**Comment by zeenix** [comment]
> I am not sure how to make it more clearer :).

How about: "found in that path" -> "re-added to the same location" and rest the same as you wrote in your comment? Also, git is not very picky on path and more about content so I'm not sure this point is important.

**Comment by GordanM** [comment]
@zeenix as per our discussion I've removed the last part to minimize the confusion.

**Pull request merged**